packages/ui/Transition/Transition.js

```diff
@@ -432,8 +434,8 @@ class Transition extends React.Component {
 			this.measureInner();
 		}
 
-		if (this.context.Subscriber) {
-			this.context.Subscriber.subscribe('resize', this.handleResize);
+		if (this.context) {
```

Comment:
When I was testing this, this.context was always valued but could be an empty object.

Reply:
Interesting, I just tested it in the unit tests. If you don't have a parent with the context I get `undefined`.